### PR TITLE
Ks 2021 02 participatory budgeting 2 phase adjustments

### PR DESCRIPTION
**Pull request by Rineee**

### CODE REVIEW

File: meinberlin/templates/a4modules/module_detail.html [hunk 1]
```diff
@@ -91,7 +91,6 @@ <h1 class="project-header__title">{{ module.name }}</h1>
         {% with phase_count=module.phases.count %}
         <div class="l-center-6">
             {% for phase in module.phases %}
-                {% if phase.is_over or phase == module.active_phase %}
```
Comment (fuzzylogic2000) [inline meinberlin/templates/a4modules/module_detail.html:94]: everything we do in the module detail, we have to also do in the project detail in many cases. https://github.com/liqd/a4-meinberlin/blob/b079c853aa08e033fd882b718159966391262161/meinberlin/apps/projects/templates/meinberlin_projects/project_detail.html#L214 
I guess putting that into includes would be a good idea... Will add an issue!

File: meinberlin/apps/budgeting/phases.py [hunk 2]
```diff
@@ -13,7 +13,8 @@ class RequestPhase(phases.PhaseContent):
     view = views.ProposalListView
 
     name = _('Request phase')
-    description = _('Participants can submit ideas, comment on and rate them.')
+    description = _('Participants can submit proposals, comment on and '
```
Comment (fuzzylogic2000) [inline meinberlin/apps/budgeting/phases.py:16]: :) Thank you!

**Review by fuzzylogic2000 — COMMENTED** [review]
Awesome! Just the module / project detail thing...

File: meinberlin/templates/a4modules/module_detail.html (hunk 1 above)
Comment (Rineee) [reply meinberlin/templates/a4modules/module_detail.html:94]: Oh yes, I forgot that the phases are shown there for single module!

**Review by fuzzylogic2000 — APPROVED** [review]
Yeah!

**Pull request merged**